<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compensateDialog</class>
 <widget class="QDialog" name="compensateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>赔偿金额(元)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>赔偿类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="adviceLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>15</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>书本原价</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LabeledEdit" name="isbnInput" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="InteractiveButtonBase" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="InteractiveButtonBase" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="InteractiveButtonBase" name="searchButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>70</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header location="global">interactive_buttons/interactivebuttonbase.h</header>
  </customwidget>
  <customwidget>
   <class>LabeledEdit</class>
   <extends>QWidget</extends>
   <header location="global">labeled_edit/labelededit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchButton</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
